CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ## [0.1.0]
+* Added test for bin heights of histograms (@ryla5068, #124)
```

Comment:
@ryla5068 i release 0.1.0 last week. so all new functionality should go in unreleased as it's not released yet. i know it's a bit confusing but generally you'll never add bullets below a release number. they will always be unreleased commits until i make a new release.